eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/RedirectClientBySubSystemHandler.java

```diff
@@ -127,22 +125,18 @@ public void handle(final HttpExchange httpExchange) throws IOException {
                 }
             } catch (Exception e) {
                 log.error("clientManage|redirectClientBySubSystem|fail|subSystem={}|destEventMeshIp"
-                    +
-                    "={}|destEventMeshPort={},errMsg={}", subSystem, destEventMeshIp, destEventMeshPort, e);
+                    + "={}|destEventMeshPort={}", subSystem, destEventMeshIp, destEventMeshPort, e);
 
```

Comment:
If you want to remove the last param of a logger, please unify the same logger in other endpoints such as `RedirectClientByPathHandler`.